primitives/src/constants/ztg.rs

```diff
@@ -2,35 +2,32 @@ use sp_runtime::Perbill;
 
 // Distribution
 
-/// Total ZTG amount for airdrops
-pub const AIRDROPS: u128 = 3_000_000;
+/// Total ZTG amount for community incentives.
+pub const COMMUNITY_INCENTIVES: u128 = 2_000_000;
 
 /// Total ZTG amount for collators
-pub const COLLATORS: u128 = 10_000_000;
-
-/// Total ZTG amount for community incentives
-pub const COMMUNITY_INCENTIVES: u128 = 2_000_000;
+pub const COLLATORS: u128 = 0;
 
 /// Total ZTG amount for liquidity mining
-pub const LIQUIDITY_MINING: u128 = 10_000_000;
+pub const LIQUIDITY_MINING: u128 = 0;
```

Comment:
I am not sure if it should be zero. Yes, during genesis on mainnet they were set to zero (this is the case for everything), nevertheless afaik we have reserved a certain amount of the complete initial ZTG pool for liquidity mining which should be reflected here.
Please verify @lsaether

Reply:
I guess we can stick with that, since that is the current state I see on the chain.

Reply:
Currently we've reserved the remaining token supply (approx 50M ZTG) in the Zeitgeist Foundation wallets, which will employ these tokens in a variety of uses including liquidity mining. However, we need to still determine the exact allocations of this pot of funds.